Makefile

```diff
@@ -162,7 +162,7 @@ root/.built: \
 	cp src/pyodide.py root/lib/python$(PYMINOR)/site-packages
 	if command -v git > /dev/null 2>&1 \
 			&& git describe --tags > /dev/null 2>&1; then \
-	sed -i "s/__version__ =.*/__version__ = '$(shell git describe --tags)'/g" \
+	sed -i "s/__version__ =.*/__version__ = '$(shell git describe --tags | sed -r 's/^v//')'/g" \
```

Comment:
I tried to save `git describe --tags | sed -r 's/^v//'` to a separate variable to reduce complexity here, but all my attempts have not been successful so far. Maybe I'm missing something about the way shell variables and makefile variables should interact.

Reply:
Yeah.  Shell scripting is always a pain.  I think this is fine for now, as I don't think it will change much.

If this starts to annoy us too much, we can write this utilities in Python, since that a hard requirement for us anyway...